### DISCUSSION TITLE
A few general questions

### DISCUSSION THREAD

**Discussion by adastx** [manna-harbour/miryoku#79] [Q&A] [answered]
I'll be building my first split keyboard soon and I'm pretty interested in using this layout but I have a few questions.

1. What is the effect of the home row mods?
I like the idea of the mods, but I am a bit worried that I am gonna be accidentally hitting them sometimes when I am typing fast. Does this happen? I practice typing as a hobby so I don't want my speed to be limited by this.
Also, do the home row mods make typing home row keys less responsive?
Ideally I don't want to notice that the mods are there.

2. Which form of colemak-dh would be best?
I switched to colemak-dh about half a year ago but I have the version where the shortcut keys in the bottom left are arranged: XCDVZ
I can see from the pictures you use the ZXCDV and I'm not sure which is actually best in terms of the colemak philosophy.
I assume this is an easy edit of the config but I'm curious what others think is the better way to go?

**Comment by manna-harbour** [comment, marked as answer]
Typing fast is fine, but there is an issue if you hold keys too long, e.g. with rolls.  Light snappy taps are best.

Taps are sent on key up, so there is a visual lag.  With clickies or tactiles you should adjust quickly to the non-visual feedback.

For hobby typing practice, a copy of the base layer without any mod taps could be used, as is commonly used for gaming.  This will be added soon, see https://github.com/manna-harbour/miryoku/issues/18.

See https://precondition.github.io/home-row-mods for more.

ZXCDV is the usual form on ergo keyboads. See https://colemakmods.github.io/mod-dh/keyboards.html#matrix-keyboards.
